configs-base/pre/buildroot/kernel

```diff
@@ -9,7 +9,6 @@ BR2_LINUX_KERNEL_NEEDS_HOST_LIBELF=y
 # see: https://kernel.org
 BR2_LINUX_KERNEL_CUSTOM_VERSION=y
 BR2_LINUX_KERNEL_CUSTOM_VERSION_VALUE="6.10.7"
-BR2_PACKAGE_HOST_LINUX_HEADERS_CUSTOM_6_8=y
```

Comment:
Could you set this to 6_10 here and/or override it wherever needed?

I remember there was some reason for setting it here, I believe it's because when buildroot is updated and a newer version becomes available, things will break unless we pin the headers version here. But this may not be the case anymore, let me know if so.

Reply:
With this line my build failed, becasuse buildroot expected 6.8 kernel headers, and got 6.10 kernel headers instead. So this is probably only needed if the kernel version is overwritten. But that would need to be tested.